plugins/lsp/default.nix

```
@@ -212,7 +212,7 @@ in
       plugins.lsp.onAttach = mkIf cfg.inlayHints ''
         -- LSP Inlay Hints {{{
         if client.server_capabilities.inlayHintProvider and vim.lsp.inlay_hint then
-          vim.lsp.inlay_hint.enable(bufnr, true)
+          vim.lsp.inlay_hint.enable(true, {})
```

```suggestion
          vim.lsp.inlay_hint.enable(true)
```

[Docs](https://neovim.io/doc/user/lsp.html#lsp-inlay_hint) say we can pass `nil` if we want filter to allow all.

---

Actually, they reference `bufnr` indented under `filters`... Maybe `bufnr` itself is now a filter (additional to [kwargs](https://neovim.io/doc/user/lua.html#kwargs))?

---

Testing now, `:lua vim.lsp.inlay_hint.enable(true, {}, 1)` seems to work:

```suggestion
          vim.lsp.inlay_hint.enable(true, {}, bufnr)
```

I guess the extra indentation is just a bug in the docs.